web-console/monitoring/alerts/alerts.md

```diff
@@ -0,0 +1,21 @@
+# Alerts
+
+## Alert Metrics
+
+![Alert details view with metrics](img/alert-metrics.png)
+- Initially the chart should be zoomed to a time period that at least includes the `for` attribute of the corresponding alert rule, but preferably 3-5 times that period as loading times allow
```

Comment:
@cshinn Should we reflect the ideal case in your example? If I'm understanding this correctly, I'm wondering if after this sentence we can say something like...For example, in the chart above, the for attribute is 5hrs so the chart should show 15-25hrs worth of time if possible. Or change your chart to use 1hr in the for attribute so you don't need to change the actual graph at all!